packages/scandipwa/src/component/CheckoutPayments/CheckoutPayments.component.js

```diff
@@ -118,7 +118,9 @@ export class CheckoutPayments extends PureComponent {
     renderPayments() {
         const { paymentMethods } = this.props;
         const { totals: { grand_total } } = this.props;
-        const paymentFree = paymentMethods.find((payment) => payment.code === 'free');
+        const codeFree = 'free';
```

Comment:
Please move this to the constant to `CheckoutPayments.config.js` file.
See other `.config.js` files as an example.